dvc/repo/experiments/executor/base.py

```diff
@@ -623,6 +627,18 @@ def _repro_dvc(  # noqa: C901
             if cls.QUIET:
                 dvc.scm_context.quiet = cls.QUIET
             old_cwd = os.getcwd()
+
+            if copy_paths:
+                for path in copy_paths:
+                    if os.path.isfile(path):
```

Comment:
Don't know if we should raise DvcException or just catch and pass here

Reply:
Do you mean if both `os.path.isfile(path)` and `os.path.isdir(path)` fail? If so, I think it makes sense to raise.

Reply:
> Do you mean if both os.path.isfile(path) and os.path.isdir(path) fail? If so, I think it makes sense to raise.

I meant if `shutil.copy/copytree` fail

Reply:
I think it should fail if either copying fails or the path can't be found. Let's assume if you are using this argument, it's important to include those paths. I think it's more useful to fail fast than find out after two long-running stages that the third fails because that file never got copied.

Reply:
Added explicit errors